.github/workflows/docker-hub.yml

```diff
@@ -5,35 +5,29 @@ on:
   push:
     branches:
       - development
-      - feature/gh-action-build-docker
+      - master
 
 jobs:
   docker:
+    if: ${{ github.repository_owner == 'fossasia' }}
```

Comment:
```suggestion
```
Is this check necessary anymore?

Reply:
That check means that, if contributors fork this repo, the copy on contributor side won't run this script.

Reply:
Perfect, makes sense